fictadvisor-api/src/v2/utils/documentation/pageText/index.ts

```diff
@@ -0,0 +1,9 @@
+import { PageTextDocumentationCreate } from './create';
+import { PageTextDocumentationGetAllTexts } from './getAllTexts';
+import { PageTextDocumentationUpdateByKey } from './updateByKey';
+
+export const PageTextDocumentation = {
+  CREATE: PageTextDocumentationCreate,
+  GET_ALL_TEXTS: PageTextDocumentationGetAllTexts,
+  UPDATE_BY_KEY: PageTextDocumentationUpdateByKey,
```

Comment:
Name of the documentation property (UPDATE_BY_KEY) must be the same as corresponding controller method (UPDATE_MANY)